nipype/utils/nipype2boutiques_example.json

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
this file should go in [`nipype/testing/data`](https://github.com/nipy/nipype/tree/master/nipype/testing/data). You can then import the data with

```python
from nipype.testing import example_data
example_data('relative/path/from/testing/directory')
```